code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -115,8 +115,6 @@
 		new /obj/item/stack/cable_coil(src)
 	if(prob(20))
 		new /obj/item/multitool(src)
-	if(prob(5))
-		new /obj/item/clothing/gloves/color/yellow(src)
```

Comment:
lame...

Reply:
oh, for a moment there, I thought this was a removal of the 5% chance for a yellow toolbox to spawn with insuls in it